code/modules/antagonists/antagonist_helpers.dm

```diff
@@ -17,7 +17,19 @@ proc/get_all_gangs()
 	for (var/datum/antagonist/gang_leader/antag_datum in get_all_antagonists(ROLE_GANG_LEADER))
 		. += antag_datum.gang
 
-	return .
+/** Returns the mind of this mind's master.
+ * If a specific antagonist role ID is provided, then the master of the antagonist datum of that ID belonging to this mind will be returned.
+ * If no antagonist role ID is provided, then the master of the newest antagonist datum will be returned.
+ */
+/datum/mind/proc/get_master(antagonist_role_id)
+	if (antagonist_role_id)
```

Comment:
what if this is called with a no id and there is no antagonists? you should set a default return value

Reply:
Is the default return value not implicitly `null`? If so, to me that seems a sensible value to return if no master could be found.

Reply:
it is, just wanted to make sure you thought about it